### PR TITLE
New min max age

### PR DESCRIPTION
**Pull request by Robertha89**
:cl: ppi
tweak: Increases the minimum and max age to 1-120 years
/:cl:

What do we want!
Higher and lower ages!
When do we want this!
Now!

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
infants as captains?

**Comment by NTSAM** [comment]
We don't need 1 year old captains thanks

File: code/__DEFINES/mob.dm [hunk 1]
```diff
@@ -18,8 +18,8 @@
 #define DROPLIMB_BLUNT 1
 #define DROPLIMB_BURN 2
 
-#define AGE_MIN 17			//youngest a character can be
-#define AGE_MAX 85			//oldest a character can be
+#define AGE_MIN 1			//youngest a character can be
+#define AGE_MAX 150			//oldest a character can be
```
Comment (tigercat2000) [inline code/__DEFINES/mob.dm]: this is distinctly not 120
and will break screams

**Comment by TullyBurnalot** [comment]
... what?

No.

Just... what?

How would you EVER possibly explain anyone under the age of 16 working here, much less at the age of ONE?!

**Comment by Robertha89** [comment]
IPC's can be 1 year old

**Comment by NTSAM** [comment]
So?

**Comment by NTSAM** [comment]
Find a way to make it work with mechanical people only then

**Comment by Magikarpet** [comment]
>literal baby/toddler/child command
Excellent for my insult game, terrible for my soul

**Comment by DisreputableSquid** [comment]
I throw my support behind this, Mostly because Diona can easily be only a few years old.

**Comment by Robertha89** [comment]
so can vox, diona, and anyone who is cloned, your new body is probably less then a year old

**Comment by tigercat2000** [comment]
lolno

**Comment by Mijhead** [comment]
maybe if limited to certain races on the lower end of spectrum

**Comment by Robertha89** [comment]
Captain's are usually one year old or younger anyway, it's lore friendly.

**Comment by NTSAM** [comment]
Screams, according to tiger, would be fucking horrendous with this

**Comment by TullyBurnalot** [comment]
For IPCs and Dionae, maybe.

Anyone else? No thanks.

**Comment by ScratKnapp** [comment]
Only if you could restrict the youngest ages to races that can plausibly only be a few years old

**Comment by Robertha89** [comment]
screams would not be too bad at 120

**Comment by NTSAM** [comment]
yeah if you call god damn helicopter screams not too bad, down with the goonscream meme 0/10

**Comment by tigercat2000** [comment]
scream at 1  https://puu.sh/nMq8J/4af4998f9d.ogg
scream at 120 https://puu.sh/nMqbK/a2995d8ee7.ogg

**Comment by FreeStylaLT** [comment]
old hag scream

**Comment by CrAzYPiLoT-SS13** [comment]
No offense, but where do you get ideas for your PRs? :-1: :x: :100:

**Comment by ProperPants** [comment]
> scream at 120 https://puu.sh/nMqbK/a2995d8ee7.ogg

This is the funniest thing I've heard in months. I'm on the verge of tears.

Old age :+1:
Young age :-1:

**Comment by NTSAM** [comment]
o yea, old age is fine but you really shouldn't lower the youngest age

**Comment by Regen1** [comment]
Oh look

Another salty PR from PPI

**Comment by Spacemanspark** [comment]
Preferably for IPCs/ diona only.

**Comment by Fox-McCloud** [comment]
No. :-1: 

screams are designed to be capped at the age of 85; you start getting higher than this and they start sounding ridiculous. No, just no.

**Comment by Necaladun** [comment]
Just no. MAYBE redoing age for various races, as pointed out for IPCs etc. If someone wanted to code that, and change the code for screams to be better suited for it, fine.

But there's absolutely no reason anyone should be able to set their age below 17. Even 17 year olds generally make a mockery of characterization.

**Comment by Crazylemon64** [comment]
Ha, and I was planning on making a JOKE pr that did this exact thing...!

**Comment by scrubmcnoob** [comment]
Bring your daughter to work day can at least be a thing now.

**Comment by TheDZD** [comment]
:no_entry_sign:

**Comment by DavidRossii** [comment]
Now I can finally play a loli.

**Comment by Spacemanspark** [comment]
Rossi no.

**Comment by ProperPants** [comment]
> screams are designed to be capped at the age of 85; you start getting higher than this and they start sounding ridiculous. No, just no

SS13 is one of the most ridiculous games in the world. Is super old age really _such_ a bad thing? It's feasible that humans could live beyond 100 with centuries of medical advancement.

**Comment by CrAzYPiLoT-SS13** [comment]
True, but will they still be able to work as efficiently as NanoTrasen
demands?

**Comment by Spacemanspark** [comment]
@CrAzYPiLoT-SS13 Sure, why not. I wouldn't be surprised.

**Comment by ProperPants** [comment]
@CrAzYPiLoT-SS13 Have you met Captain Stumpy? Because he's a really horrible example for old age and competence (god do I love that captain, though).

**Comment by CrAzYPiLoT-SS13** [comment]
Unfortunately, I haven't. I don't know, I have changed my stance to neutral
on the old age. Young age is still a no-no.

**Comment by KasparoVy** [comment]
Crazylemon64's suggestion makes loads of sense. Maybe raise the age cap for humans, leave it as-is for everyone else-- maybe drop it a tiny bit for Unathi because they grow up (for the most part) in pretty trash living conditions, etc. etc. Reflecting life spans and such.

**Comment by Robertha89** [comment]
species based age sounds neat

**Pull request closed without merging**